data/python/python3-scipy-test.py

```diff
@@ -8,7 +8,11 @@
 import math
 
 import numpy as np
-from scipy.integrate import quad, simps
+from scipy.integrate import quad
```

Comment:
Please add a comment for your future selves, e.g.
```suggestion
# scipy.integrate.simp is deprecated but still used on older SLES versions
from scipy.integrate import quad
```

Reply:
Done!